api/views.py

```diff
@@ -22,3 +22,18 @@ def post(self, request):
             return Response(status=status.HTTP_201_CREATED)
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+class CustomerList(APIView):
+    authentication_classes = [TokenAuthentication, HostAuthentication, SignatureValidation]
```

Comment:
You'll need to rebase onto main and use the same authentication class of `DeliveryList`, only `SignatureValidation`.

Reply:
Yes, PR is getting conflict files.